controllers/userController.js

```diff
@@ -0,0 +1,26 @@
+const userService = require("../services/userService");
+
+const signup = async (req, res) => {
+  try {
+    const { email, password, name, birth, phoneNumber, address } = req.body;
+    if (!email || !password || !name || !birth || !phoneNumber) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+    await userService.signup(
+      email,
+      password,
+      name,
+      birth,
+      phoneNumber,
+      address
+    );
+    return res.status(200).json({ message: "SUCCESS_SIGNUP" });
```

Comment:
```suggestion
    return res.status(201).json({ message: "SUCCESS_SIGNUP" });
```
데이터를 생성 혹은 수정하는 것이기 때문에 더 적절한 status code로 바꿔줄 수 있을 것 같습니다.